src/lib/content.js

```diff
@@ -200,9 +218,13 @@ function attachColorLine(profiles) {
     const found = profiles.find(item => { return item.profile === profileName });
     const color = found && found.color || null;
 
+    var label = usernameMenu.querySelector('.nav-elt-label');
+    if (found && found.image) {
+        label.insertAdjacentHTML('beforebegin', Sanitizer.escapeHTML(`<img id="AESW_Image" src=${found.image.replace(/"/g, '')} width="14em" height="14em" style="float: left; padding-right: 1em">`));
```

Comment:
The width and height attributes of img support only **px** as unit, so add the style attribute like `style="width: 1em; height: 1em"` to it.